### ISSUE TITLE
Does not work in macOS sierra 10.12

### ISSUE THREAD

**Issue by tempura666** [josh-/CloudyTabs#37]
After update from El Capitan,
There is no reaction when click menubar icon.

**Comment by ianbannerman** [comment]
I'm seeing this as well.

**Comment by GitHub2016X** [comment]
Josh, CloudyTabs provides something incredibly useful, especially to users like me who prefer a default browser that isn't Safari. Is it even possible to make the app work with Sierra? If so, are you planning to revise it?

**Comment by brianvanburken** [comment]
I experienced the same thing when upgraded to macOS Sierra version 10.12 during the beta and now the final. Though this must be a problem on Apple's side. Since it is working correctly in for me in macOS version 10.12.1 beta. So hold on till the release of version 10.12.1! 😉

**Comment by zigg** [comment]
@brianvanburken I'm also on the latest 10.12.1 public beta and have an unresponsive menu bar icon. I just installed CloudyTabs today.

**Comment by tempura666** [comment]
@brianvanburken I also 10.2.1 public beta 4 was installed.
However, although it can be said through from beta 1 to 4, The problem is not resolved.

Perhaps, CloudyTabs will need to support for Sierra.

**Comment by brianvanburken** [comment]
Then I stand corrected. I experienced the same bug described in the this ticket. But then I installed the beta (currently on build `16B2548a`) and it started working normal again. With this my assumption was that Apple fixed something in the beta that was broken before.

**Comment by valentinocossar** [comment]
Same here.

**Comment by Frohmen** [comment]
I'm also having this problem with macOS Sierra not running beta

**Comment by shanegriffiths** [comment]
Same here :(

**Comment by josh-** [comment]
Hey all, please see [this comment](https://github.com/josh-/CloudyTabs/issues/33#issuecomment-260865431) in #33. Sadly this project won't be able to function until we work out how to overcome Apple's syncing changes in 10.12

**Comment by ilovecookiez11** [comment]
![screen shot 2017-07-08 at 1 08 12 pm](https://user-images.githubusercontent.com/17679753/27988059-b7cc6a22-63de-11e7-9b93-c5256d77d7f5.png)

I give this the official "Works on my machine!" Seal of approval.
It never stopped working on my end.
If anything, I'm mostly affected by the URL encoding bug, which could be easily sorted out if anyone figured out the right encoding format.

**Comment by josh-** [comment]
I don't know how that's possible @ilovecookiez11 but very interesting!

Would you be able to say which version you're running? The URL encoding issue should have been fixed in [version 1.5](https://github.com/josh-/CloudyTabs/releases/tag/v1.5)

**Comment by ilovecookiez11** [comment]
Hi @josh- ,
I'm using 1.6, but I'm not sure if it's your fork or cl3m's, though.
Also com.apple.Safari.plist doesn't look encrypted for me at all.
![screen shot 2017-07-16 at 5 05 05 pm](https://user-images.githubusercontent.com/17679753/28251726-10996e78-6a49-11e7-993b-49500a90e8c1.png)

**Comment by josh-** [comment]
What versions of iOS/macOS are your other devices running @ilovecookiez11?

**Comment by ilovecookiez11** [comment]
@josh- I'm using 10.2 on my iPhone SE and 10.12.5 on my MacBook.

**Comment by toddbluhm** [comment]
Just wanted to give a little input on this. I run a fork of cloudy tabs, but ran into the same encryption issue. I noticed that for whatever reason, cloudy tabs worked for me for along time. The safari data was unencrypted and easily readable, even after OS updates. Then when I reformatted my mac a few months back, all the safari data that synced back up was encrypted and thus cloudy tabs broke :(

@ilovecookiez11 my question to you would be, when was the last time you reformatted?

**Comment by ilovecookiez11** [comment]
@toddbluhm Around July-August 2015. When I bought my MacBook it came with no OS so I downloaded 10.8 with the Internet Recovery option. Right after that, I downloaded Mavericks from the App Store, but then Yosemite came out, so I updated to it in October 2015. Now everytime a new version comes out I just upgrade it through the App Store like most people do. I'm holding on High Sierra, though. Haven't had the need to format my drive at all.
